requirements.txt

```diff
@@ -0,0 +1,50 @@
+astroid==2.4.2
+attrs==20.3.0
+blis==0.7.4
+catalogue==2.0.1
+certifi==2020.12.5
+chardet==4.0.0
+click==7.1.2
+cymem==2.0.5
+en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0-py3-none-any.whl
+Flask==1.1.2
+gunicorn==20.0.4
+idna==2.10
+iniconfig==1.1.1
+isort==5.7.0
+itsdangerous==1.1.0
+Jinja2==2.11.3
+lazy-object-proxy==1.4.3
+MarkupSafe==1.1.1
+mccabe==0.6.1
+Morfessor==2.0.6
+murmurhash==1.0.5
+numpy==1.20.0
+packaging==20.9
+pathy==0.3.5
+plac==1.1.3
+pluggy==0.13.1
+polyglot==16.7.4
+preshed==3.0.5
+py==1.10.0
+pycld2==0.41
+pydantic==1.7.3
+PyICU==2.6
+pylint==2.6.0
+pyparsing==2.4.7
+pytest==6.2.2
+requests==2.25.1
+six==1.15.0
+smart-open==3.0.0
+spacy==3.0.3
+spacy-legacy==3.0.1
+srsly==2.4.0
+thinc==8.0.1
+toml==0.10.2
+tqdm==4.56.0
+typer==0.3.2
+urllib3==1.26.3
+wasabi==0.8.2
+Werkzeug==1.0.1
+wrapt==1.12.1
+xx-ent-wiki-sm @ https://github.com/explosion/spacy-models/releases/download/xx_ent_wiki_sm-3.0.0/xx_ent_wiki_sm-3.0.0-py3-none-any.whl
```

Comment:
same as above; separate in another auto-generated file at build time, and concatenate it to `requirements.txt` when building docker image.